### PR TITLE
fix: Dsync bugs

### PR DESCRIPTION
**Pull request by joeauyeung**
## What does this PR do?

This PR aims to fix some dsync bugs that were missed in the reviews

Fixes # (issue)

## Requirement/Documentation

- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- Are there environment variables that should be set?
- What are the minimal test data to have?
- What is expected (happy path) to have (input and output)?
- Any other important info that could help to test that PR

## Mandatory Tasks

- [ ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/trpc/server/routers/viewer/dsync/delete.handler.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ export const deleteHandler = async ({ ctx, input }: Options) => {
     });
   }
 
-  await prisma.dSyncData.delete({
+  await prisma.dSyncData.findMany({
```
Comment (Udit-takkar) [inline packages/trpc/server/routers/viewer/dsync/delete.handler.ts:30]: what is the user of findMany here ?

Comment (CarinaWolli) [reply packages/trpc/server/routers/viewer/dsync/delete.handler.ts:30]: Also not sure what `findMany` does here, then we don't delete anything? Would `deleteMany` solve it? that shouldn't throw an error if it doesn't exist

Comment (joeauyeung) [reply packages/trpc/server/routers/viewer/dsync/delete.handler.ts:30]: The dsync data still exists in the SAML DB which is handled with the Jackson package.

Comment (joeauyeung) [reply packages/trpc/server/routers/viewer/dsync/delete.handler.ts:30]: I was blanking so hard on this. It was supposed to be deleteMany

**Review by CarinaWolli — APPROVED** [review]

**Pull request merged**